src/kixi/web_server.clj

```diff
@@ -1,16 +1,47 @@
 (ns kixi.web-server
-  (require [clojure.java.io :as io]
+  (:require [clojure.java.io :as io]
            [bidi.bidi :refer [tag]]
            [bidi.vhosts :refer [make-handler vhosts-model]]
            [com.stuartsierra.component :as component]
-           [taoensso.timbre :as timbre :refer [infof]]
-           [yada.yada :as yada :refer [resource]]
+           [taoensso.timbre :as timbre :refer [infof info error]]
+           [yada.yada :as yada]
+           [yada.resource :as yr]
            [yada.consume :refer [save-to-file]]
-           [yada.resources.webjar-resource :refer [new-webjar-resource]]))
+           [yada.resources.webjar-resource :refer [new-webjar-resource]])
+  (:require [kixi.metrics :as metrics]))
 
 (defn say-hello [ctx]
+  (info "Saying hello")
   (str "Hello " (get-in ctx [:parameters :query :p]) "!\n"))
 
+(defn yada-timbre-logger
+  [ctx]
+  (when (= 500 (get-in ctx [:response :status]))
```

Comment:
Ah yeah, there's the that error predicate. Yeah sure, this was a cheap way of filtering. Getting to this point was the trouble! :)